doc/source/design/board/gd32vf103c_longan_nano.rst

```diff
@@ -20,6 +20,22 @@ Is is similar to the well known STM32-based "Blue Pill" board.
 
     Sipeed Longan Nano Board.
 
+
+.. _design_board_sipeed_longan_nano_versions:
+
+Versions
+~~~~~~~~
+
+There are two versions of this board available.
+
+* GD32VF103C\ **B**\ T6 with 128k Flash / 32k RAM
+
+* GD32VF103C\ **8**\ T6 with 64k Flash / 20k RAM. This is sometimes called the "lite" version.
```

Comment:
The PR #18 will introduce the `VARIANT` variable, so you can add some note here to let user know how to select the version developer want to use.